### DISCUSSION TITLE
Failing to deploy edge functions "Error: Relative import path "events" not prefixed with / or ./ or ../_"

### DISCUSSION THREAD

**Discussion by justinekatebe** [supabase/supabase#21139] [Questions] [answered]
I've been trying to deploy functions from my computer for a while and I have been failing. This is the error I keep getting:

Error: Error bundling function: exit status 1
file:///src/import_map.json
file:///src/index.ts
error: Uncaught (in promise) Error: Relative import path "events" not prefixed with / or ./ or ../ and not in import map from "https://esm.sh/v135/@types/node@18.16.19/events.d.ts"
      const ret = new Error(getStringFromWasm0(arg0, arg1));
                  ^
    at __wbg_new_8d2af00bc1e329ee (https://deno.land/x/eszip@v0.30.0/eszip_wasm.generated.js:312:19)
    at <anonymous> (https://deno.land/x/eszip@v0.30.0/eszip_wasm_bg.wasm:1:79439)
    at <anonymous> (https://deno.land/x/eszip@v0.30.0/eszip_wasm_bg.wasm:1:1388039)
    at <anonymous> (https://deno.land/x/eszip@v0.30.0/eszip_wasm_bg.wasm:1:1862894)
    at __wbg_adapter_18 (https://deno.land/x/eszip@v0.30.0/eszip_wasm.generated.js:146:6)
    at real (https://deno.land/x/eszip@v0.30.0/eszip_wasm.generated.js:130:14)

Try rerunning the command with --debug to troubleshoot the error.

Anybody here that can help?

**Comment by oliverrees** [comment, marked as answer]
Getting same error. If I add `&no-check` to `import { createClient } from "https://esm.sh/@supabase/supabase-js@2";` then it allows me to deploy the function but it's not an ideal solution...

**Reply by justinekatebe** [reply]
This helped! Thank you so much!

**Reply by vicman** [reply]
hey there, where do I add the &no-check attribute?

**Reply by justinekatebe** [reply]
<img width="567" alt="Screenshot 2024-05-22 at 16 29 29" src="https://github.com/supabase/supabase/assets/82674078/ed7a4cef-7cf9-4de3-82c5-f3a7746f5716">

Put it at the end of the url like this

**Comment by bombillazo** [comment]
Are you using Deno v2? I am getting the same error now that I started using Deno v2
